playground/main.js

```diff
@@ -60,7 +60,15 @@ const renderPlaygroundWidget = (options = {}) => {
     }
   );
 
+  signIn.on('ready', () => {
+    console.log('===== playground widget ready event received =====');
```

Comment:
I am guessing these console log statements are only so that we can do a t.getBrowserConsoleMessages() in the individual tests ?

Reply:
sure. 
I was thinking people may or may not pay attention to comment ;) and if they do change it, test would failed.